static/js/components/LimitSelector.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { Dispatch, FC, SetStateAction } from "react";
+import { Col, Input, RadioInput, Row } from "@canonical/react-components";
+import { UNITS, MemoryLimit } from "../panels/ProfileForm";
+
+interface Props {
+  cpuLimit: string;
+  setCpuLimit: Dispatch<SetStateAction<string>>;
+  memoryLimit: MemoryLimit;
+  setMemoryLimit: Dispatch<SetStateAction<MemoryLimit>>;
+}
+
+const LimitSelector: FC<Props> = ({
+  cpuLimit,
+  setCpuLimit,
+  memoryLimit,
+  setMemoryLimit,
+}) => {
+  const cpuLimitHelp = (
+    <>
+      See{" "}
+      <a href="https://linuxcontainers.org/lxd/docs/latest/reference/instance_options/#cpu-limits">
```

Comment:
Should add `target="_blank" rel="noreferrer"` to the link.

We should change the link text, see https://heyoka.medium.com/dont-use-click-here-f32f445d1021
Maybe "See docs for the allowed formats." and link on the "allowed formats"?